### PR TITLE
feat(NightDesign): added night.design basic presence

### PR DESCRIPTION
**Pull request by BCCSTeam**
## Description 
feat(NightDesign): added night.design basic presence
fix(NightDesign): removed localhost url

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
RmFKU
wWuhU

</details>

### CODE REVIEW

File: websites/N/NightDesign/presence.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	const pathSegments: string[] = path.split("/").filter(segment => segment);
+	for (let i = 0; i < pathSegments.length; i++) {
+		const pathSegement: string = pathSegments
+				.slice(0, pathSegments.length - i)
+				.join("/"),
+			data = strings[pathSegement];
+
+		if (!data) continue;
+
+		switch (pathSegement) {
```
Comment (RisingSunLight42) [inline websites/N/NightDesign/presence.ts:51]:
```suggestion
		switch (pathSegment) {
```

**Review by RisingSunLight42 — CHANGES_REQUESTED** [review]
Also shorten your queries and avoid to put a big part of code in a for, it cost a lot and can be done in another way.

Also delete package-lock.json from your PR.

File: websites/N/NightDesign/metadata.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	"service": "NightDesign",
+	"description": {
+		"en": "NightDesign is a Minecraft Cosmetics and Social-Network Website"
+	},
+	"url": [
+		"night.design",
+		"dev.night.design"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/A9934OD.png",
```
Comment (DarkVillager) [inline websites/N/NightDesign/metadata.json:16]: This needs to be 512x512

File: websites/N/NightDesign/presence.ts [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+		server: "$0 Server Profile",
+
+		team: "Team List",
+
+		stateResources: "Type: $0 | Creator: $1",
+	},
+	start = Math.round(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/A9934OD.png",
```
Comment (DarkVillager) [inline websites/N/NightDesign/presence.ts:24]: Same here

File: websites/N/NightDesign/presence.ts [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+				if (pathSegments.length !== 2) {
+					presenceData.details = strings.resources;
+					continue;
+				}
+
+				presenceData.details = strings.resource.replace(
+					"$0",
+					textContent(
+						"body > app-root > div.router-content > app-resource > div.page-header > div > div > div.col-md-4 > ul > li:nth-child(4) > a"
+					)
```
Comment (DarkVillager) [inline websites/N/NightDesign/presence.ts:62]: Try to shorten the selectors like this:
```suggestion
document.querySelector('[class="breadcrumb justify-content-end"]')?.lastChild?.textContent
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]
Also fix the changes to package-lock.json

if you have questions or sum lmk

**Pull request closed without merging**